src/test/java/io/github/bookcrawler/core/impl/packtparser/PacktBookInfoParserTest.java

```diff
@@ -0,0 +1,47 @@
+package io.github.bookcrawler.core.impl.packtparser;
+
+import io.github.bookcrawler.core.impl.SourceScrappingResult;
+import io.github.bookcrawler.entities.BookInfo;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PacktBookInfoParserTest {
+
+    @Test
+    public void testParsingBookElementsFromDocument() throws IOException {
+
+        //given
+        PacktBookInfoParser packtBookInfoParser = new PacktBookInfoParser();
+        SourceScrappingResult sourceScrappingResult = mock(SourceScrappingResult.class);
+        Document source = Jsoup.parse(new File("src/test/java/io/github/bookcrawler/core/" +
```

Comment:
`Profile` and `JsoupScrapper` classes are not a part of my changes :octopus: Here I'm handling `IOException` by throwing it in method signature :ghost:, the other case is that this is a test only and I always expect a file in this location. 

The `IOException` from `JsoupScrapper` class is handleld by returning empty `List` of books and then we could handle it on higher levels, like ex. show some message on website. 

Thanks for review. :+1: